fixtures/search-syntax/boolean_filter_passthrough.json

```diff
@@ -1,13 +1,13 @@
 [
   {
-    "query": "project_id:1",
```

Comment:
Just making sure - `project_id` and `issue.id` are being replaced in these fixtures due to they actually not being valid?

Reply:
@malwilley  They're actually valid but we're deprecating them and decided to not list them in the fields at all.